packages/dialect-react-ui/utils/addressesUtils.ts

```diff
@@ -31,9 +31,15 @@ export const getUsersCount = (
     .filter((address) => address.address.verified)
     .filter((address) => addressTypePredicate(address.address.type));
 
-  // If there're no notifications type set up for this dapp, return all addresses count
+  // If there're no notifications type set up for this dapp, return all unique users count
   if (!subscriptions.length && notificationTypeId === null) {
-    return enabledAndVerifiedAddresses.length;
+    return [
+      ...new Set(
```

Comment:
`Array.from(new Set(...)).length` is also good.

Reply:
Or even better: `new Set().size`. So no need to convert to array to get the length.